client/a8c-for-agencies/sections/purchases/payment-methods/payment-method-add/payment-method-form/index.tsx

```diff
@@ -192,6 +192,11 @@ function PaymentMethodForm() {
 						returnQueryArg
 					)
 				);
+			} else if ( returnQueryArg?.includes( 'new_dev_site' ) ) {
+				const params = {
+					add_new_dev_site: true,
```

Comment:
For consistency: What about if we used the same `payment_method_added` parameter as is being used above?

```suggestion
					payment_method_added: true,
```

I think it would also make the code where the parameter is being extracted more readable:

```js
const paymentMethodAdded = getQueryArg( window.location.href, 'payment_method_added' );
```

vs 

```js
const paymentMethodAdded = getQueryArg( window.location.href, 'add_new_dev_site' );
```

Reply:
Yes, you are right. But I suggest we go with `add_new_dev_site` on all places instead `payment_method_added` for one reason.
There are other teams working on that codebase. If they eventually create another flow, that redirects to the `/sites` page after adding a credit card, this could trigger the demo site modal.

Reply:
That’s a valid point. When I was thinking about it, I felt like the following was a bit confusing:

```js
const paymentMethodAdded = getQueryArg( window.location.href, ‘add_new_dev_site’ );
```

But maybe we could keep it as it is, just to be safe.